release-notes/releases-index.json

```diff
@@ -13,7 +13,11 @@
             "eol-date": "2026-05-11",
             "release-type": "sts",
             "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json",
-            "supported-os.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/supported-os.json"
+            "patch-releases-info-uri": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json",
+            "patch-releases-index-uri": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/patch-releases-index.json",
+            "supported-os.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/supported-os.json",
```

Comment:
Dupe property.

```suggestion
```

Reply:
Are you saying it is a dupe with line 19? Line 15 and line 18 have to be kept for compat.

Reply:
Sorry - this was me being an idiot and reading the diff incorrectly 🤦

Reply:
All good.